anaconda_anon_usage/tokens.py

```diff
@@ -24,6 +27,41 @@ def version_token():
     return __version__
 
 
+@cached
+def system_token():
+    """
+    Returns the system/organization token. Unlike the other
+    tokens, it is desirable for this token to be stored in
+    a read-only/system location, presumably installed
+    The system/organization token can be stored anywhere
+    in the standard conda search path. Ideally, an MDM system
+    would place it in a read-only system location.
+    """
+    # Do not import SEARCH_PATH directly since we need to
+    # temporarily patch it for testing
+    for path in c_constants.SEARCH_PATH:
```

Comment:
I would suggest to use [`Configuration._expand_search_path(c_constants.SEARCH_PATH)`](https://github.com/conda/conda/blob/1fc28363416cf1163c4425d6840d08123055fce1/conda/common/configuration.py#L1389-L1412) here, to expand the search paths without affecting the running conda process, for forward-compatibility mostly around expansion. I haven't fully checked if the static method is available in all support conda versions of aau though. Maybe in that case vendor it?

Reply:
Good idea, I'll investigate. I originally did not want to do that because I had some other custom treatments in mind. But I walked those back so this should be feasible

Reply:
We can't use it, unfortunately, because it will only return directories where `.condarc`, `condarc`, or `condarc.d` already exists. So in other words, if `/etc/conda/org_token` exists but not `/etc/conda/.condarc`, then the `Configuration` code won't find it.

Reply:
Ah, gotcha, bummer